Resources/Private/CodeTemplates/Git/config

```diff
@@ -0,0 +1,6 @@
+[core]
+	repositoryformatversion = 0
+	filemode = true
+	bare = false
+	logallrefupdates = true
+	precomposeunicode = true
```

Comment:
Hm... one remark here.
git for windows generated this config for a blank repo
```
[core]
	repositoryformatversion = 0
	filemode = false
	bare = false
	logallrefupdates = true
	symlinks = false
	ignorecase = true
```

Reply:
https://git-scm.com/docs/git-config

* `filemode = false`- fine setting it to false (not apply executable bit)
* `symlinks = false` - fine setting it to false (since symlinks are not required inside a package)
* `ignorecase = true` - fine setting it to true (having `File.html` and `file.html` would cause trouble in general between different operating systems)
* `precomposeunicode = true` - could stay (since it only affects Git for macOS)